torchao/quantization/quant_primitives.py

```diff
@@ -136,7 +136,7 @@ def _get_reduction_params(block_size, input_size):
 
 def quantize_affine(
     input: torch.Tensor,
-    block_size: List[int],
+    block_size: Tuple[int, ...],
```

Comment:
Why this change?

Reply:
I think a Tuple is better for the block_size argument, since it's immutable